interfaces/builtin/udisks2.go

```diff
@@ -0,0 +1,920 @@
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.UDisks2.Drive
+    member=SetConfiguration
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.UDisks2.Drive
+    member=PowerOff
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.DBus.Properties
+    member=Get
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.DBus.Properties
+    member=PropertiesChanged
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+# Drive.Ata
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.UDisks2.Drive.Ata
+    member=Smart{Update,GetAttributes,SelftestStart,SelftestAbort,SetEnabled}
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.UDisks2.Drive.Ata
+    member=Pm{GetState,Standby,Wakeup}
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/drives/*
+    interface=org.freedesktop.UDisks2.Drive.Ata
+    member=SecurityEraseUnit
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+# UDisks2 MDRaid interface
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/mdraid/*
+    interface=org.freedesktop.UDisks2.MDRaid
+    member={Start,Stop}
+    peer=(label=###PLUG_SECURITY_TAGS###),
+    
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/mdraid/*
+    interface=org.freedesktop.UDisks2.MDRaid
+    member={Add,Remove}Device
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/mdraid/*
+    interface=org.freedesktop.UDisks2.MDRaid
+    member=SetBitmapLocation
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/mdraid/*
+    interface=org.freedesktop.UDisks2.MDRaid
+    member=RequestSyncAction
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/mdraid/*
+    interface=org.freedesktop.DBus.Properties
+    member=Get
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/mdraid/*
+    interface=org.freedesktop.DBus.Properties
+    member=PropertiesChanged
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+# Udisks2 Job interface
+dbus (receive)
+    bus=system
+    path=/org/freedesktop/UDisks2/jobs/*
+    interface=org.freedesktop.UDisks2.Job
+    member=Cancel
+    peer=(label=###PLUG_SECURITY_TAGS###),
+
+dbus (send)
+    bus=system
+    path=/org/freedesktop/UDisks2/jobs/*
+    interface=org.freedesktop.UDisks2.Job
+    member=Completed
+    peer=(label=###PLUG_SECURITY_TAGS###),
```

Comment:
Ok, rather than listing "here too" everywhere, you can condense by dropping 'member' and condense per interface. Or you can let everything in along these lines:
```
dbus (receive, send)
    bus=system
    path=/org/freedesktop/UDisks2/**
    interface=org.freedesktop.UDisks2.*
    peer=(label=###PLUG_SECURITY_TAGS###),
```

Again, it just depends on the API you want to support. In general it makes sense to go this fine-grained when:
- you are trying to carve out unprivileged APIs and privileged APIs (eg, location-observe vs location-control)
- you only want to support a subset of the API

The policy you've specified is all highly privileged and it appears you are listing most, if not all of the API here. If you want to support the whole API, make it easy on yourself and future maintenance with glob rules.